SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverRPC.cpp

```diff
@@ -0,0 +1,471 @@
+													   TUniquePtr<TRPCQueue<FRPCPayload, FSpatialGDKSpanId>>& QueuePtr)
+{
+	// TODO
+}
+
+void USpatialNetDriverRPC::MakeRingBufferWithACKReceiver(FName ReceiverName, ERPCType RPCType, Worker_ComponentSetId AuthoritySet,
+														 TUniquePtr<TRPCBufferReceiver<FRPCPayload, TimestampAndETWrapper>>& ReceiverPtr)
+{
+	// TODO
+}
+
+USpatialNetDriverServerRPC::USpatialNetDriverServerRPC() {}
+
+void USpatialNetDriverServerRPC::FlushRPCUpdates()
+{
+	USpatialNetDriverRPC::FlushRPCUpdates();
+	FlushRPCQueue(*ClientReliableQueue);
+	FlushRPCQueue(*ClientUnreliableQueue);
+}
+
+void USpatialNetDriverServerRPC::ProcessReceivedRPCs()
+{
+	USpatialNetDriverRPC::ProcessReceivedRPCs();
+
+	auto CanExtractRPCs = [this](Worker_EntityId EntityId) {
+		return CanExtractRPCOnServer(EntityId);
+	};
+	auto ProcessRPC = [this](Worker_EntityId EntityId, ReceivedRPC RPCData, const FRPCMetaData& MetaData) {
+		return ApplyRPC(EntityId, RPCData, MetaData);
+	};
+
+	ServerReliableReceiver->ExtractReceivedRPCs(CanExtractRPCs, ProcessRPC);
+	ServerUnreliableReceiver->ExtractReceivedRPCs(CanExtractRPCs, ProcessRPC);
+}
+
+void USpatialNetDriverServerRPC::GetRPCComponentsOnEntityCreation(const Worker_EntityId EntityId, TArray<FWorkerComponentData>& OutData)
+{
+	USpatialNetDriverRPC::GetRPCComponentsOnEntityCreation(EntityId, OutData);
+
+	{
+		RPCWritingContext Ctx(ClientReliableQueue->Name, MakeDataWriteCallback(OutData));
+		ClientReliableQueue->Flush(EntityId, Ctx, StandardQueue::SentRPCCallback(), /*bIgnoreAdded*/ true);
+	}
+	{
+		RPCWritingContext Ctx(ClientUnreliableQueue->Name, MakeDataWriteCallback(OutData));
+		ClientUnreliableQueue->Flush(EntityId, Ctx, StandardQueue::SentRPCCallback(), /*bIgnoreAdded*/ true);
+	}
+}
+
+void USpatialNetDriverServerRPC::Init(USpatialNetDriver* InNetDriver, const SpatialGDK::FSubView& InActorAuthSubView,
+									  const SpatialGDK::FSubView& InActorNonAuthSubView)
+{
+	USpatialNetDriverRPC::Init(InNetDriver, InActorAuthSubView, InActorNonAuthSubView);
+	const USpatialGDKSettings* Settings = GetDefault<USpatialGDKSettings>();
+
+	auto constexpr RequiredAuth = SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID;
+
+	MakeRingBufferWithACKSender("ClientReliable", ERPCType::ClientReliable, RequiredAuth, ClientReliableSender, ClientReliableQueue);
+	MakeRingBufferWithACKSender("ClientUnreliable", ERPCType::ClientUnreliable, RequiredAuth, ClientUnreliableSender,
+								ClientUnreliableQueue);
+
+	MakeRingBufferWithACKReceiver("ServerReliable", ERPCType::ServerReliable, RequiredAuth, ServerReliableReceiver);
+	MakeRingBufferWithACKReceiver("ServerUnreliable", ERPCType::ServerUnreliable, RequiredAuth, ServerUnreliableReceiver);
+
+	{
+		auto RPCType = ERPCType::NetMulticast;
+		auto RPCDesc = RPCRingBufferUtils::GetRingBufferDescriptor(RPCType);
+
+		// TODO
+	}
+}
+
+USpatialNetDriverClientRPC::USpatialNetDriverClientRPC() {}
+
+void USpatialNetDriverClientRPC::FlushRPCUpdates()
+{
+	USpatialNetDriverRPC::FlushRPCUpdates();
```

Comment:
Super?

Reply:
Not a UObject anymore, so Super is not available.